### DISCUSSION TITLE
GitHub theme not applying to HTML pages?

### DISCUSSION THREAD

**Discussion by leojamesstrong** [community/community#23161] [Other Feature Feedback, Questions, & Ideas]
<p>I’ve been trying to apply the Minima theme to my GitHub Pages site but it doesn’t seem to be working. I’ve changed the theme in the Settings and I’ve added this code to the top of my page(s) HTML:</p>
<pre><code class="lang-auto">---
layout: /_layouts/_config
title: Home
---
</code></pre>
<p>When I first committed the changes, the page layout changed, but after I refreshed the page, the changes were no longer in effect and the page had no theme (instead just looking like the basic HTML). Is there a way I can fix this?</p>

**Comment by parkr** [comment]
<p>Hi <a class="mention" href="/leojamesstrong">@leojamesstrong</a>! I’m sorry you’re having trouble using the Minima theme.</p>
<p>The first thing to try here is a different value for <code>layout</code>. The <code>layout:</code> key in your YAML front matter should <a href="https://jekyllrb.com/docs/step-by-step/04-layouts/#use-layouts" rel="noopener nofollow ugc">be the name of the layout</a>, not a path. The minima theme has <a href="https://github.com/jekyll/minima/tree/v2.5.1/_layouts" rel="noopener nofollow ugc">four layouts you can choose from</a>:</p>
<ul>
<li>
<code>default</code> – base layout, sets up the header, footer, and a container</li>
<li>
<code>home</code> – this is the layout the minima theme recommends for your <code>index.html</code> file</li>
<li>
<code>page</code> – use this for pages like <code>about.md</code>
</li>
<li>
<code>post</code> – use this for everything in <code>_posts/*</code>
</li>
</ul>
<p>For your Home example above, try changing the layout to <code>layout: home</code> and see if that fixes things. It should show you the basic Minima home page as seen in the README file.</p>
<p>Hope this helps!</p>

**Comment by leojamesstrong** [comment]
<p>Hello,<br>
That solution kind of worked? When I changed one of my pages’ layouts to <code>post</code> it seemed to work, but using <code>home</code> on my index HTML page doesn’t seem to have worked. Thanks for your help!</p>

**Comment by otter-computer** [comment, marked as answer]
<p>It looks like there was a mix up with the theme you’re using. <a class="mention" href="/parkr">@parkr</a> gave you the link for the <a href="https://github.com/jekyll/minima"><code>Minima</code></a> theme but it looks like you’ve actually got the <a href="https://github.com/pages-themes/minimal"><code>Minimal</code></a> theme set for this repository.</p>
<p>The available layouts for this theme are <code>default</code> and <code>post</code>:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/pages-themes/minimal/tree/master/_layouts" target="_blank" rel="noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:400/400;"><img src="https://avatars.githubusercontent.com/u/22329739?s=400&amp;amp;v=4" class="thumbnail" width="400" height="400"></div>

<h3><a href="https://github.com/pages-themes/minimal/tree/master/_layouts" target="_blank" rel="noopener">pages-themes/minimal</a></h3>

<p><a href="https://github.com/pages-themes/minimal/tree/master/_layouts" target="_blank" rel="noopener">master/_layouts</a></p>

  <p><span class="label1">Minimal is a Jekyll theme for GitHub Pages. Contribute to pages-themes/minimal development by creating an account on GitHub.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>This means that setting <code>layout: home</code> for your <code>index.html</code> page won’t render with a template. To fix this you’ll need to set <code>layout: default</code> in the front matter of your <code>index.html</code> file.</p>

**Comment by leojamesstrong** [comment]
<p>Thank you so much, that worked!</p>
